src/utils/helper.js

```diff
@@ -1,11 +1,30 @@
 import fs from 'fs';
 import path from 'path';
+import toml from 'toml';
+
+function readAndParseTomlConfig(filePath) {
+    try {
+        const content = fs.readFileSync(filePath, 'utf-8');
+        return toml.parse(content);
+    } catch (err) {
+        console.error('Error reading or parsing TOML file:', err);
+        process.exit(-1);
+    }
+}
+
+//Checking if the directory exists and if not, create it
```

Comment:
Can you also remove the function `checkAndCreateDirectory` since it's not being used anymore? :) @mnajibi

Reply:
Resolved in commit [bf64cbbc](https://github.com/sdthaker/Node-TILify/pull/13/commits/bf64cbbc3675e3e8c77d8e1e0d7a73e1841bff47)